projects/components/src/table/data/table-cdk-data-source.ts

```diff
@@ -89,10 +90,14 @@ export class TableCdkDataSource implements DataSource<TableRow> {
         const key = keyValueTuple[0];
         const value = keyValueTuple[1];
 
+        const columnConfig = this.columnConfigs.get(key)!;
+        const parser = new columnConfig.parser!();
```

Comment:
Rather than new'ing this everywhere, can we just put an instance of the parser on the column config?